.github/workflows/Default.yml

```diff
@@ -0,0 +1,38 @@
+name: Log4JNA Default Build
+
+on:
+  push:
+    branches: [master]

```

Comment:
I think we could build this on all branches, would use `*`.

Reply:
resolved in #54